src/mapbox/mapbox.js

```diff
@@ -149,7 +149,11 @@ export default class Mapbox {
   // (e.g. until "componentDidUpdate")
   resize() {
     this._map.resize();
-    this._map._render();
+    try {
```

Comment:
Is it possible to instead check if map style is loaded before calling `_render`?